guide/08 Finding places with Geocoding/Understanding geocoders.ipynb

```diff
@@ -1037,9 +1037,9 @@
  "metadata": {
   "anaconda-cloud": {},
   "kernelspec": {
-   "display_name": "Python [Root]",
+   "display_name": "Python [conda env:arcgis-pyapi]",
```

Comment:
There are no real changes to this notebook, so it should be excluded from the pull request. Also, we shouldn't change the kernelspec to include custom kernels as those opening notebooks with non-standard/default kernels are prompted with an error saying that the kernel cannot be found and if another kernel should be used.